src/main/resources/migrations/00056_index_transaction_metadata_value_column.sql

```diff
@@ -0,0 +1,6 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_index_transaction_metadata_value_column runInTransaction:false
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS transaction_metadata_value_column_idx
```

Comment:
Can we rename index to suggest lower similar to `transaction_lower_reference_idx` ?

```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS transaction_metadata_lower_value_idx
```